docs/user/status.md

```diff
@@ -3,6 +3,27 @@
 [The ClusterVersion object](../dev/clusterversion.md) sets `conditions` describing the state of the cluster-version operator (CVO).
 This document describes those conditions and, where appropriate, suggests possible mitigations.
 
+## Failing
+
+When `Failing` is True, the CVO is failing to reconcile the cluster with the desired release image.
+In all cases, the impact on the cluster will be that later nodes in [the manifest graph](reconciliation.md#manifest-graph) may not be [reconciled](reconciliation.md#reconciling-the-graph).
```

Comment:
` that later nodes` does not communicate clearly.  Can we just say some nodes?

Reply:
How about "dependent nodes"?

Reply:
Works for me

Reply:
Fixed "later nodes" -> "dependent nodes" with 89b9af1 -> d459a57.